acme_metapackage.bash

```diff
@@ -46,8 +29,8 @@ conda metapackage -c conda-forge -c acme -c uvcdat -c opengeostat acme-unified $
 "cartopy  ${OPERATOR}0.15.1" \
 "progressbar ${OPERATOR}2.3" \
 "scipy <1.0.0" \
-"matplotlib" \
-"basemap" \
```

Comment:
matplotlib and basemap get added via acme_diags, I assume?

Reply:
Well, `matplotlib` is a dependency of `acme_diags`. As for `basemap`, why did we remove that @doutriaux1?

Reply:
We definitely need both matplotlib and basemap for MPAS-Analysis.  Is there any harm in leaving them in here?

Reply:
it won't hurt anything, since they're already brought in by acme_diags.